### PR TITLE
Expose refresh policy 

### PR DESCRIPTION
**Pull request by liatiusim**

### CODE REVIEW

File: package/src/main/resources/etc/custom.system.properties [hunk 1]
```diff
@@ -113,6 +113,8 @@ org.apache.unomi.profile.purge.inactiveTime=${env:UNOMI_PROFILE_PURGE_INACTIVETI
 org.apache.unomi.profile.purge.existTime=${env:UNOMI_PROFILE_PURGE_EXISTTIME:--1}
 # Purge all sessions/events that have been created for a specific number of months
 org.apache.unomi.event.purge.existTime=${env:UNOMI_EVENT_PURGE_EXISTTIME:-12}
+# Persistence policy of events when they are save int the persistence service. Can be: none / block_until_ready / force_readiness
```
Comment (giladw) [inline package/src/main/resources/etc/custom.system.properties:116]: typo: int the persistence...

File: plugins/baseplugin/src/main/java/org/apache/unomi/plugins/baseplugin/actions/SetEventOccurenceCountAction.java [hunk 2]
```diff
@@ -115,10 +122,11 @@ public int execute(Action action, Event event) {
             toDateTime = LocalDateTime.ofInstant(toDateCalendar.toInstant(), ZoneId.of("UTC"));
         }
 
-        LocalDateTime eventTime = LocalDateTime.ofInstant(event.getTimeStamp().toInstant(),ZoneId.of("UTC"));
-
-        if (inTimeRange(eventTime, numberOfDays, fromDateTime, toDateTime)) {
-            count++;
+        if (eventService.getEventPersistencePolicy().equals(PersistencePolicy.NONE.getPolicy())) {
```
Comment (giladw) [inline plugins/baseplugin/src/main/java/org/apache/unomi/plugins/baseplugin/actions/SetEventOccurenceCountAction.java:125]: what is being done here?

Comment (liatiusim) [reply plugins/baseplugin/src/main/java/org/apache/unomi/plugins/baseplugin/actions/SetEventOccurenceCountAction.java:125]: IWe want to increase the counter only if the current event is not returned from the query (which will probably happen when the policy is none - basically it's for backward compatibilty)

File: api/src/main/java/org/apache/unomi/api/services/EventService.java [hunk 3]
```diff
@@ -85,6 +85,8 @@
      */
     List<EventProperty> getEventProperties();
 
+    String getEventPersistencePolicy();
```
Comment (lyogev) [inline api/src/main/java/org/apache/unomi/api/services/EventService.java:88]: It's not related to the service...

File: package/src/main/resources/etc/custom.system.properties [hunk 4]
```diff
@@ -113,6 +113,8 @@ org.apache.unomi.profile.purge.inactiveTime=${env:UNOMI_PROFILE_PURGE_INACTIVETI
 org.apache.unomi.profile.purge.existTime=${env:UNOMI_PROFILE_PURGE_EXISTTIME:--1}
 # Purge all sessions/events that have been created for a specific number of months
 org.apache.unomi.event.purge.existTime=${env:UNOMI_EVENT_PURGE_EXISTTIME:-12}
+# Persistence policy of events when they are saved in the persistence service. Can be: none / block_until_ready / force_readiness
+org.apache.unomi.event.persistence.policy=${env:UNOMI_EVENT_PERSISTENCE_POLICY:-none}
```
Comment (lyogev) [inline package/src/main/resources/etc/custom.system.properties:117]: It should be under elasticsearch, no?

**Pull request closed without merging**